devenv.nix

```
@@ -178,7 +178,18 @@ in {
       example = [ "mailparse" ];
     };
 
+    additionalMysqldSettings = lib.mkOption {
+      type = lib.types.attrsOf lib.types.str;
+      default = { };
+      description = "Additional mysqld settings";
+      example = {
+         lower_case_table_names = "1";
```

Please change the example, f.e.:
```suggestion
         sync_binlog = "0";
```


Please do not advertise `lower_case_table_names`. This can lead to errors for existing installations. Additionally, this is no default value or general recommended one.